Ginger/Ginger/SolutionWindows/AccessibilityRulePage.xaml.cs

```diff
@@ -0,0 +1,147 @@
+            mSolution = WorkSpace.Instance.Solution;
+            string allProperties = string.Empty;
+            WorkSpace.Instance.PropertyChanged += WorkSpacePropertyChanged;
+            LoadGridData();
+            SetAppsGrid();
+            mAccessibilityConfiguration = WorkSpace.Instance.SolutionRepository.GetFirstRepositoryItem<AccessibilityConfiguration>();
+            mAccessibilityConfiguration.DefaultExcludeRule = mAccessibilityConfiguration.DefaultExcludeRule != null ? mAccessibilityConfiguration.DefaultExcludeRule : new ();
+        }
+        private void WorkSpacePropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == nameof(WorkSpace.Solution))
+            {
+                mSolution = WorkSpace.Instance.Solution;
+                LoadGridData();
+            }
+        }
+
+        private void SetAppsGrid()
+        {
+            xAccessibilityRulesGrid.SetGridEnhancedHeader(Amdocs.Ginger.Common.Enums.eImageType.Accessibility, $"{GingerCore.General.GetEnumValueDescription(typeof(eTermResKey), nameof(eTermResKey.AccessibilityRules))}", saveAllHandler: SaveHandler, null, true);
+            GridViewDef view = new GridViewDef(GridViewDef.DefaultViewName);
+            view.GridColsView = new ObservableList<GridColView>();
+            view.GridColsView.Add(new GridColView()
+            {
+                Field = nameof(AccessibilityRuleData.Active),
+                WidthWeight = 2,
+                StyleType = GridColView.eGridColStyleType.Template,
+                CellTemplate = (DataTemplate)FindResource("CheckBoxTemplate"),
+                AllowSorting = true
+            });
+            view.GridColsView.Add(new GridColView()
+            {
+                Field = nameof(AccessibilityRuleData.RuleID),
+                Header = "RuleID",
+                ReadOnly = true,
+                WidthWeight = 15,
+                AllowSorting = true
+            });
+            view.GridColsView.Add(new GridColView()
+            {
+                Field = nameof(AccessibilityRuleData.Description),
+                Header = "Description",
+                WidthWeight = 40,
+                ReadOnly = true
+            });
+            view.GridColsView.Add(new GridColView()
+            {
+                Field = nameof(AccessibilityRuleData.Impact),
+                Header = "Severity",
+                WidthWeight = 5,
+                ReadOnly = true,
+                AllowSorting = true
+            });
+            view.GridColsView.Add(new GridColView()
+            {
+                Field = nameof(AccessibilityRuleData.Tags),
+                WidthWeight = 30,
+                ReadOnly = true
+            });
+
+            xAccessibilityRulesGrid.AddLabel("Note: We will analyze the active items from the list below for accessibility. Testing and deactivating items won't be taken into account for accessibility testing.");
```

Comment:
Change it to "Note: Ginger will only analyze the active accessibility testing rules; inactive rules will not be considered."

Reply:
Done